marblecutter/mosaic.py

```diff
@@ -55,7 +59,8 @@ def composite(sources, (bounds, bounds_crs), (height, width), target_crs):
 
         data, _ = window_data
         if data.mask.any():
-            # mask outliers
+            # mask outliers (intended for DEM boundaries)
+            LOG.info("masking outliers")
```

Comment:
I'd kinda like to keep it around in case it triggers on RGB imagery, 'cause I'm not entirely sure what sort of output it'll produce.